cvat-ui/src/components/quality-control/styles.scss

```diff
@@ -172,6 +172,10 @@ $excluded-background: #d9d9d973;
     .ant-tabs-content-holder, .ant-tabs-content, .ant-tabs-tabpane {
         height: 100%;
     }
+
+    .ant-tabs-tabpane-active{
```

Comment:
Please, consider my [comment here](https://github.com/cvat-ai/cvat/issues/9158#issuecomment-2694744628).
Hiding overflow will not make any good to us as some information will be clipped